corehq/apps/userreports/expressions/__init__.py

```diff
@@ -2,6 +2,15 @@
 from django.utils.module_loading import import_string
 from corehq.apps.userreports.expressions.factory import ExpressionFactory
 
+
+def get_custom_ucr_expressions():
+    custom_ucr_expressions = settings.CUSTOM_UCR_EXPRESSIONS
+
+    for expression_list in settings.CUSTOM_UCR_EXPRESSION_LISTS:
+        custom_ucr_expressions += import_string(expression_list)
```

Comment:
ah, i see. the two levels of dynamic import indirection and variable names were confusing me. I think we should probably try and tidy it up a little bit (for one there is no need to do dynamic imports if the module where the expression functions are defined is defining the list), but makes more sense now.

I think the point about the adding to the list without copying still holds.